fdbclient/VersionVector.h

```diff
@@ -43,6 +43,14 @@ struct VersionVector {
 		maxVersion = version;
 	}
 
+	void setVersion(const std::set<Tag>& tags, Version version) {
```

Comment:
Extracted from Dan's PR.

Reply:
you can rebase as PR #5114 has merged. That may fix the conflict as well.

Reply:
Ok.